lib/src/managers/data_manager.dart

```diff
@@ -100,4 +111,96 @@ abstract class DataManager<T, P> {
 
   @protected
   Future<Map<String, T>> fetch(P params);
+
+  Future<void> initializeCache() async {
+    if (kIsWeb) {
+      return;
```

Comment:
why web is not supported? :(

Reply:
Old package didnt support web, forgot to remove if